cmd/config/config.go

```diff
@@ -102,6 +102,14 @@ func GetDefaultDNSSyncConfig(nt nodeconfig.NetworkType) harmonyconfig.DnsSync {
 	return dnsSync
 }
 
+func GetDefaultLocalnetConfig() harmonyconfig.LocalnetConfig {
+	localnetConfig := harmonyconfig.LocalnetConfig{
+		BlocksPerEpoch:   nodeconfig.GetDefaultLocalnetBlocksPerEpoch(),
+		BlocksPerEpochV2: nodeconfig.GetDefaultLocalnetBlocksPerEpochV2(),
```

Comment:
what will be the required changes if one day we decided to no longer have BlocksPerEpochV2 ?

Reply:
this will be deprecated then

Reply:
or even a BlocksPerEpochV3 ?

Reply:
This PR helps any new configurations simply can be added later by add a few line of code